requirements-ci.txt

```diff
@@ -1,4 +1,4 @@
-pennylane>=0.30
+git+https://github.com/PennyLaneAI/pennylane.git
```

Comment:
Revert this before merge

Reply:
and by revert you mean bump to 0.32 : )

Reply:
LOL, you just want me to do the pre-release bump for you

Reply:
I want you to not get experimental changes for 0.33 from master : )